packages/compass-e2e-tests/helpers/commands/drop-collection-from-sidebar.ts

```diff
@@ -4,48 +4,24 @@ import * as Selectors from '../selectors';
 export async function dropCollectionFromSidebar(
   browser: CompassBrowser,
   connectionName: string,
-  dbName: string,
+  databaseName: string,
   collectionName: string
 ): Promise<void> {
-  const connectionId = await browser.getConnectionIdByName(connectionName);
-
-  // search for the collecton in the sidebar filter
-  await browser.clickVisible(Selectors.SidebarFilterInput);
-  await browser.setValueVisible(
-    Selectors.SidebarFilterInput,
-    `^(${dbName}|${collectionName})$`
+  await browser.selectCollectionMenuItem(
+    connectionName,
+    databaseName,
+    collectionName,
+    'drop-collection'
   );
-  const dbElement = await browser.$(
-    Selectors.sidebarDatabase(connectionId, dbName)
-  );
-  await dbElement.waitForDisplayed();
+  await browser.dropNamespace(collectionName);
 
-  // wait for the collection to become displayed
+  // wait for it to be gone
+  const connectionId = await browser.getConnectionIdByName(connectionName);
   const collectionSelector = Selectors.sidebarCollection(
     connectionId,
-    dbName,
+    databaseName,
     collectionName
   );
-  await browser.scrollToVirtualItem(
-    Selectors.SidebarNavigationTree,
-    collectionSelector,
-    'tree'
-  );
   const collectionElement = await browser.$(collectionSelector);
-  await collectionElement.waitForDisplayed();
-
-  // open the drop collection modal from the sidebar
-  await browser.hover(collectionSelector);
-
-  // NOTE: if the menu was already open for another collection this could get
-  // confusing. Also this selector is just for the actions button and it is
-  // assumed that at this point it is the only one. But the drop confirmation
-  // usually catches that.
-  await browser.clickVisible(Selectors.SidebarNavigationItemShowActionsButton);
```

Comment:
This wasn't scoped to the collection. If one collection was active it would show the three dot menu. If you then hover over a second collection it would show that three dot menu too..